.travis.yml

```diff
@@ -1,8 +1,15 @@
 sudo: false
 language: node_js
 node_js:
-  - '5'
-  - '4'
-  - '0.10'
+  - "0.10"
+  - "0.12"
+  - "4"
+  - "5"
+  - "iojs"
```

Comment:
This isn't my choice. It's what @shama wanted and we have in grunt-contrib-internal.